versions/gui_gui.py

```diff
@@ -0,0 +1,101 @@
+text_label_column=0
+copy_button_column=1
+text_output_column=2
+dir = os.path.dirname(__file__)
+
+def copy_to_clipboard(text):
+    pyperclip.copy(text['text'])
+
+def add_text(*args):
+    new_text = translate_text.get()
+    x_size=int(8.5*len(new_text)+240)
+    root.geometry(f"{x_size}x{350}")
+    reversed_text['text'] = converter.reverse_text(new_text)
+    flipped_text['text'] = converter.text_flip(new_text)
+    enchanted_text['text'] = converter.enchant_text(new_text)
+    capitalized_text['text'] = converter.case_switch(new_text, "upper")
+    lowercased_text['text'] = converter.case_switch(new_text, "lower")
+    leetspeak_text['text'] = converter.leetspeak(new_text)
+    scrambled_text['text'] = converter.scramble_text(new_text)
+    piglatin_text['text'] = converter.piglatin(new_text)
+    zalgo_text['text'] = converter.zalgo_text(new_text)
+    emoticons_text['text'] = converter.text_to_emoticons(new_text)
+
+def add_copy_buttons():
+    ttk.Button(root, command=partial(copy_to_clipboard, reversed_text), image = copy_icon).grid(sticky = W, row=1, column=copy_button_column)
+    ttk.Button(root, command=partial(copy_to_clipboard, flipped_text), image = copy_icon).grid(sticky = W, row=2, column=copy_button_column)
+    ttk.Button(root, command=partial(copy_to_clipboard, enchanted_text), image = copy_icon).grid(sticky = W, row=3, column=copy_button_column)
+    ttk.Button(root, command=partial(copy_to_clipboard, capitalized_text), image = copy_icon).grid(sticky = W, row=4, column=copy_button_column)
+    ttk.Button(root, command=partial(copy_to_clipboard, capitalized_text), image = copy_icon).grid(sticky = W, row=5, column=copy_button_column)
+    ttk.Button(root, command=partial(copy_to_clipboard, leetspeak_text), image = copy_icon).grid(sticky = W, row=6, column=copy_button_column)
+    ttk.Button(root, command=partial(copy_to_clipboard, scrambled_text), image = copy_icon).grid(sticky = W, row=7, column=copy_button_column)
+    ttk.Button(root, command=partial(copy_to_clipboard, piglatin_text), image = copy_icon).grid(sticky = W, row=8, column=copy_button_column)
+    ttk.Button(root, command=partial(copy_to_clipboard, zalgo_text), image = copy_icon).grid(sticky = W, row=9, column=copy_button_column)
+    ttk.Button(root, command=partial(copy_to_clipboard, emoticons_text), image = copy_icon).grid(sticky = W, row=10, column=copy_button_column)
+
+root = Tk()
+root.title("Translator")
+ico = Image.open(os.path.join(dir,'../_internal/images/icon.ico'))
+photo = ImageTk.PhotoImage(ico)
+copy_icon = Image.open(os.path.join(dir,'../_internal/images/content_copy.png'))
+copy_icon = ImageTk.PhotoImage(copy_icon)
+root.wm_iconphoto(False, photo)
+root.geometry("240x350")
+
+translate_text = StringVar()
+translate_text.trace_add("write", add_text)
+input_widget = ttk.Entry(root, textvariable=translate_text)
+input_widget.grid(row=0, column=0, columnspan=2)
+
+ttk.Label(text="Reversed:").grid(row=1,column=text_lable_column)
```

Comment:
Starting from this line, you're copy pasting a lot over per label, which is unnecessary. You could put this in a `while` loop since they seem to all be the same except for variable names, text labels and rows

Reply:
So how would I get it to change the variable name?